server/test/services/ti/TrustedIntermediaryServiceTest.java

```diff
@@ -300,4 +303,40 @@ private void setupTIAccount(
     applicantData.setDateOfBirth(dob);
     applicant.save();
   }
+
+  @Test
+  public void updateApplicantDateOfBirth_throwsApplicantNotFoundException() {
+    Http.RequestBuilder requestBuilder =
+        addCSRFToken(fakeRequest().bodyForm(ImmutableMap.of("dob", "2022-07-07")));
+    Form<UpdateApplicantDob> form =
+        formFactory.form(UpdateApplicantDob.class).bindFromRequest(requestBuilder.build());
+    assertThrows(
+        ApplicantNotFoundException.class, () -> service.updateApplicantDateOfBirth((long) 0, form));
+  }
+
+  @Test
+  public void updateApplicantDateOfBirth_unformattedDate() throws ApplicantNotFoundException {
+    Http.RequestBuilder requestBuilder =
+        addCSRFToken(fakeRequest().bodyForm(ImmutableMap.of("dob", "2022-20-20")));
+    Form<UpdateApplicantDob> form =
+        formFactory.form(UpdateApplicantDob.class).bindFromRequest(requestBuilder.build());
+    Account account = repo.lookupAccountByEmail("email30").get();
+    Form<UpdateApplicantDob> returnedForm = service.updateApplicantDateOfBirth(account.id, form);
+    assertThat(returnedForm.hasErrors()).isTrue();
+    assertThat(returnedForm.error("dob").get().message())
+        .isEqualTo("Date of Birth must be in MM/dd/yyyy format");
+  }
+
+  @Test
```

Comment:
Mind adding a test for the case called out above where the DOB is attempted to be updated for an account not tied to the trusted intermediary?

Reply:
I removed that check as that case is kind of impossible. I would like to pick your brains, if you think its possible.

Reply:
Redone as we discussed yesterday. Testcase: updateApplicantDateOfBirth_throwsApplicantNotFoundExceptionDueToIncorrectTIGroup is added.